src/components/pricing/GrowthPricingTier.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Button, TextField } from 'components';
+
+import TierName from './TierName';
+import TierDescription from './TierDescription';
+import TierBulletsSection from './TierBulletsSection';
+
+const GrowthPricingTier = () => (
+  <div className="border border-gray-200 rounded-lg shadow-sm divide-y divide-gray-200">
+    <div className="p-6">
+      <TierName name="Growth" />
+      <TierDescription description="For excellent engineering orgs who want to maintain effectiveness through hypergrowth" />
+
+      <p className="text-4xl font-extrabold text-gray-900 mb-14">Custom</p>
+
+      <div className="mb-10">
+        <TextField
```

Comment:
Does this necessarily need to be a text field if they will be disclosing numbers in the demo anyways. 
Also are we alienating customers that want SLA and support etc. but don't have an org worth of 150 seats yet when we use the text '150 developers or more'?

Reply:
Let's continue this conversation on the clubhouse ticket.